lib/cli.py

```diff
@@ -263,7 +264,6 @@ def load_filter(self):
         filter_files = list(filter(lambda fn: Path(fn).exists(), filter_files))
         
         if filter_files:
-            import_FaceFilter()
```

Comment:
I still don't get why this would cause issues. I couldn't replicate but can test again tomorrow. 

By moving this to the top of load_filter, it means dlib will be launched for this on Extract regardless of whether we're filtering faces or not as self.process_arguments has:

`self.filter = self.load_filter()`

with no check on whether it's needed.

There may be a bug in the load_filter code though:

`        if filter_files:`
`            import_FaceFilter()`
`            print('Loading reference images for filtering: %s' % filter_files)`
`            return FaceFilter(filter_files, nfilter_files, self.arguments.ref_threshold)`

It looks like FaceFilter only gets called if there is a value for filter_files. If there is a value for nfilter_files and no value for filter_files then the FaceFilter doesn't get called :/

Unless I'm missing something, I think the check should probably be:
`if filter_files or nfilter_files:`

with my import_FaceFilter() remaining where I put it and the print statement updated to reflect the change.

I have never used this functionality, so may be off the mark